Makefile

```
@@ -1,7 +1,23 @@
-CPPFLAGS += -I /usr/lib/kframework/include -I dummy-version -I plugin -I plugin-c -I libff/build/install/include -I deps/cpp-httplib
-CXX=clang++-8
+# set default install prefix if not set
+PREFIX ?= $(CURDIR)/build
 
-.PHONY: build
+K_RELEASE ?= $(dir $(shell which kompile))..
```

If you switch to the following logic it will (in order of priority):

-   Use whatever `K_RELEASE` is set to in environment.
-   Use submodule if present.
-   Use global install if present.

```suggestion
K_SUBMODULE := $(DEPS_DIR)/k
ifneq (,$(wildcard $(K_SUBMODULE)/k-distribution/target/release/k/bin/*))
    K_RELEASE ?= $(abspath $(K_SUBMODULE)/k-distribution/target/release/k)
else
    K_RELEASE ?= $(dir $(shell which kompile))..
endif
```

This makes sure that we're using the submodule when building this for experimenting.

---

Ah nevermind, I see that we're just using a global install of K for this one anyway. So your approach should be good because it will let us set `K_RELEASE` to the correct submodule in KEVM, for example, and then get the correct behavior.